doozerlib/cli/release_gen_assembly.py

```diff
@@ -48,7 +53,11 @@ def releases_gen_assembly(ctx, name):
               help="When using --auto-previous, set custom suggestions URL, load from {major}-{minor}-{arch}.yaml")
 @pass_runtime
 @click.pass_context
-def gen_assembly_from_releases(ctx, runtime: Runtime, nightlies: Tuple[str, ...], standards: Tuple[str, ...], custom: bool, in_flight: Optional[str], previous_list: Tuple[str, ...], auto_previous: bool, graph_url: Optional[str], graph_content_stable: Optional[str], graph_content_candidate: Optional[str], suggestions_url: Optional[str]):
+def gen_assembly_from_releases(ctx, runtime: Runtime, nightlies: Tuple[str, ...], standards: Tuple[str, ...],
```

Comment:
Same as above. I am assuming the prototype hasn't changed, but it's hard to tell just by looking at this diff